src/mmw/js/src/modeling/gwlfe/runoff/views.js

```diff
@@ -154,6 +154,16 @@ function monthSorter(a, b) {
 
 window.monthSorter = monthSorter;
 
+function SumFormatter(data) {
```

Comment:
This is defined as capitalized `SumFormatter` instead of the regular `sumFormatter` because otherwise jslint gives the following error:

```
> kj npm run lint
+ ARGS='run lint'
+ vagrant ssh app -c 'cd /opt/app && envdir /etc/mmw.d/env npm run lint'

> model-my-watershed@0.0.1 lint /opt/app
> jshint $(find js -name *.js -not -path '*/shim/*')

js/src/modeling/gwlfe/runoff/views.js: line 158, col 17, Possible strict violation.

1 error
```

for this line:

```js
var field = this.field,
```

See here for details: http://stackoverflow.com/a/27117322/2053314